cmd/search.go

```diff
@@ -54,33 +53,28 @@ func (o *SearchOptions) Complete(f Factory, args []string) (err error) {
 // Run executes the search command
 func (o *SearchOptions) Run() (err error) {
 
-	if o.Reindex {
-		printInfo(o.Out, "building index...")
-		done := false
-		if err = o.SearchRequests.Reindex(&core.ReindexSearchParams{}, &done); err != nil {
-			return err
-		}
-		printSuccess(o.Out, "reindex complete")
-	}
+	// TODO: add reindex option back in
 
-	p := &repo.SearchParams{
-		Q:      o.Query,
-		Limit:  30,
-		Offset: 0,
+	p := &core.SearchParams{
+		QueryString: o.Query,
+		Limit:       100,
+		Offset:      0,
 	}
-	res := []repo.DatasetRef{}
+	results := &[]core.SearchResult{}
 
-	if err = o.SearchRequests.Search(p, &res); err != nil {
+	if err = o.SearchRequests.Search(p, results); err != nil {
```

Comment:
I think it's better to have `results` be a `[]core.SearchResult{}` instead of a pointer-to `[]core.SearchResult`, as there's some danger that others will use `results` later in this function in a way where the pointer messes things up. Instead I'd prefer we do this:
```golang
results := []core.SearchResult{}
if err = o.SearchRequests.Search(p, &results); err != nil {
// ...
```